.circleci/config.yml

```diff
@@ -69,6 +71,8 @@ jobs:
           only_for_branches: <<pipeline.parameters.only_for_branches>>
   integration-test-windows:
     executor: win/default
+    environment:
+      GOFLAGS: -mod=vendor
```

Comment:
I accidentally deleted these lines in this commit https://github.com/lacework/go-sdk/commit/5aeec3c51184fc7e43e1c9dc413d256c98b8c516

![tenor-25002146](https://user-images.githubusercontent.com/5712253/103795105-ce865980-5002-11eb-99ff-8be5609379c9.gif)